src/components/atoms/SfCheckbox/SfCheckbox.html

```diff
@@ -0,0 +1,26 @@
+<div class="sf-checkbox">
+  <input
+    :id="id"
+    :name="name"
+    type="checkbox"
+    class="sf-checkbox__field"
+    :value="true"
+    v-model="isChecked"
+    :disabled="disabled" />
+  <!-- @slot Custom checkbox label markup (binds `checked` boolean) -->
+  <slot name="label" :checked="isChecked">
+    <label :for="id" class="sf-checkbox__label">
+      <span class="sf-checkbox__icon">
+        <!-- @slot Custom checked icon image markup (binds `checked` boolean) -->
+        <slot name="icon" :checked="isChecked">
+          <img
+            class="sf-checkbox__icon-svg"
+            src="/assets/check.svg"
+            alt="Check icon" />
+        </slot>
+      </span>
+      <!-- @slot Custom checkbox label text (binds `checked` boolean) -->
+      <slot :checked="isChecked" />
```

Comment:
I guess this slot should have a name

Reply:
Do you have a name suggestion? This slot may be used in almost all cases, and I think the others will almost never be used, so I left it as the default.

Reply:
Maybe label?